### PR TITLE
fd is now officially in Debian

### PR DESCRIPTION
**Pull request by sylvestre**
Work by @paride and the debian rust team!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,15 @@ Download the latest `.deb` package from the [release page](https://github.com/sh
...
 ```
 
+### On Ubuntu
+
+If you run Debian sid (and soon Debian Buster), fd is
+[officially maintained by Debian](https://tracker.debian.org/pkg/rust-fd-find).
+```
+$ sudo apt-get install fd-find
+```
+The binary is called `fdfind` as the binary name `fd` is already used by another package.
```
Comment (sharkdp) [inline README.md]: I wonder if we should recommend `alias fd=fdfind` here to make the documentation (README and man page) consistent?

Comment (sylvestre) [reply README.md]: yeah, probably. 
we are also discussing about other options like install fd in a specific path and recommending to update PATH

**Comment by sylvestre** [comment]
by the way @sharkdp how would you fill about renaming fd to fdfind?
it would help on debian :)

**Comment by sharkdp** [comment]
> by the way @sharkdp how would you fill about renaming fd to fdfind?
> it would help on debian :)

I would rather not. I don't really like `fdfind`/`fd-find` because it duplicates the word "find" and its abbreviation.

We have had a similar discussion in [this ticket](https://github.com/sharkdp/fd/issues/42). However, this discussion is now over a year old and I believe that the name "fd" has now made it into a few online discussions and blog posts. "fd" is also available as a package named "fd" (with a binary named "fd") in a lot of different distributions and package managers. See [this ticket](https://github.com/sharkdp/fd/issues/317#issuecomment-419555117) for a similar discussion with @paride. I would therefore like to avoid any renaming.

See also [my comment in this discussion](https://github.com/sharkdp/fd/issues/104#issuecomment-336244635):

> I knew that there was a price to pay (name clashes, harder to find) when naming this `fd`, but I wanted to have something short like `rg`, `ag`, `ls`, etc. Of course, I could have asked all users to set an `alias fd`, but somehow that doesn't seem very user-friendly to me.

**Comment by sharkdp** [comment]
Also, sorry for the trouble that this causes with Debian packaging. I really appreciate your effort.

Does Ubuntu have a different policy?

**Comment by paride** [comment]
@sharkdp I'm not sure, but Ubuntu is likely to inherit the Debian package at some point...

**Pull request merged**